src/EnergyPlus/WindowManager.cc

```diff
@@ -5081,8 +5306,9 @@ namespace WindowManager {
             // Interaction with shade or blind, if one of these is present, is ignored. See below for
             // separate calculation of shade/blind temperature.
 
-            rguess(1) = 1.0 / (state.dataWindowManager->hcout + hrad);
-            rguess(state.dataWindowManager->nglface + 1) = 1.0 / (state.dataWindowManager->hcin + hrad);
+            rguess(1) = 1.0 / (state.dataWindowManager->hcout + hrad) * state.dataHeatBalSurf->CoeffAdjRatioOut(SurfNum);
```

Comment:
1.0 / ((state.dataWindowManager->hcout + hrad) * state.dataHeatBalSurf->CoeffAdjRatioOut(SurfNum)) ?

Reply:
Yes, there's a missing parenthesis. Fixed it.